tool/aligner_utils.py

```diff
@@ -348,10 +348,17 @@ def _bwa_aln_sai(self, genome_file, reads_file, params):  # pylint: disable=no-s
             Location of the reads file in the file system
         params : dict
             Dictionary of the parameters for bwa aln
+        single : bool
+            True for single ended, will use 4 threads for processing
+            False for paired end, will use 2 threads for processing
```

Comment:
Just out of curiosity, why single ends gets 4 threads?? thanks

Reply:
This is due to the setup of the VMs. There are 4 threads available. For the single ended data it can use all 4 threads. For the paired end there is multithreading to process both files at the same time, so they can only use 2 threads each